src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/EndAppealConfirmation.java

```diff
@@ -45,12 +57,16 @@ public PostSubmitCallbackResponse handle(
                 + "Contact the respondent to tell them what has changed, including any action they need to take.\n"
             );
         } else {
-
             postSubmitResponse.setConfirmationHeader("# You have ended the appeal");
-            postSubmitResponse.setConfirmationBody(
-                "#### What happens next\n\n"
-                + "A notification has been sent to all parties.<br>"
-            );
+
```

Comment:
The logic here is a little too nested and complicated, see if you can simplify it

Reply:
I've refactored to code to make them clearer to read, hope that's okay.